src/screenshot_gui.cpp

```diff
@@ -72,3 +72,21 @@ void ShowScreenshotWindow()
 	CloseWindowById(WC_SCREENSHOT, 0);
 	new ScreenshotWindow(&_screenshot_window_desc);
 }
+
+/**
+ * Hides or shows the screenshot window when taking a screenshot.
+ * @param hidden Are we hiding the window or showing it again after the screenshot is taken?
```

Comment:
`hidden` confuses me. Does it mean the window was hidden, but you are unhiding it, or does it mean it will become hidden? In general, you want to use active forms for function names and parameters there in. So `hide` would be much more clear.

Similar, the function name is misleading. You are not setting it "hidden". You set the visibility of the window. So `SetScreenshotWindowVisibility` would be a lot more verbose.